backend/typescript/utilities/errorUtils.ts

```diff
@@ -10,3 +10,5 @@ export class NotFoundError extends Error {
     this.name = "NotFoundError";
   }
 }
+
+export const INTERNAL_SERVER_ERROR_MESSAGE = "Internal server error occured.";
```

Comment:
since you are adding this constant, can we update behaviourRoute to also use this